Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -154,9 +154,11 @@ public func verifySnapshot<Value, Format>(
 
       if !attachments.isEmpty {
         #if !os(Linux)
-        XCTContext.runActivity(named: "Attached Failure Diff") { activity in
-          attachments.forEach {
-            activity.add($0)
+        if ProcessInfo.processInfo.environment.keys.contains("__XCODE_BUILT_PRODUCTS_DIR_PATHS") {
```

Comment:
Relying on a private Xcode implementation detail :/

Reply:
There are a few other keys we could rely on instead, like `XCTestConfigurationFilePath`...